src/transformers/models/mistral/modeling_mistral.py

```diff
@@ -1006,6 +1007,7 @@ def forward(
                 (batch_size, seq_length),
                 inputs_embeds,
                 past_key_values_length,
+                sliding_window=self.config.sliding_window if is_torch_version_greater_or_equal_than_2_2_0 else None,
```

Comment:
if the argument does not exist in previous versions, this will fail no? We should have a `kwarg` instead no?

Reply:
> if the argument does not exist in previous versions, this will fail no? We should have a `kwarg` instead no?

in the function definition listed below,
https://github.com/huggingface/transformers/blob/main/src/transformers/modeling_attn_mask_utils.py#L336, there is a default None here, so i think this will not fail here in previous versions. @ArthurZucker

Reply:
Ah sorry sorry you are completely right!